### PR TITLE
fix: changes for grammatical and spelling errors

### PR DESCRIPTION
**Pull request by HimanshuNarware**
feat:  grammatical and spelling errors were resolved from the website, It will improve the readability of the website as well as improve the user experience

### Related Issue 

Closes: #
#772

#### Changes made 👷🏻‍♂️
resolved all grammatical and spelling errors present on the website and other files

### CODE REVIEW

File: public/CODE_OF_CONDUCT.md
Comment (tamalCodes) [inline public/CODE_OF_CONDUCT.md:1]: Hi there, please donot make any changes to this file, as we are working on it.

File: rules/FrontendSetup.md
Comment (tamalCodes) [inline rules/FrontendSetup.md:1]: Hi there, why is `# Frontend Setup` done 2 times and why is the whole thing copied and pasted again ? 
The goal was to change particular words that might have been grammatically incorrect but here the whole thing was just copied and pasted. Please remove this.

image

File: src/components/Events/index.jsx [hunk 1]
```diff
@@ -21,8 +21,8 @@ const EventsBanner = () => {
       >
         <h1 className="mb-4">Check out our events!!</h1>
         <p className="donate-details">
-          All our partnered NGOs , hosts various events be it educational,
-          cleaning mother earth, funding events, health camps and many more !!
+        All our partnered NGOs, host various events be they educational,
```
Comment (tamalCodes) [inline src/components/Events/index.jsx:24]: Please change `they` back to `it` as both are grammatically correct.

File: src/components/Events/index.jsx [hunk 2]
```diff
@@ -21,8 +21,8 @@ const EventsBanner = () => {
...
+          cleaning mother earth, funding events, health camps, and many more  !!
         </p>
```
Comment (tamalCodes) [inline src/components/Events/index.jsx:26]: Remove the duplicate space after `more` and before `!!`

File: src/pages/Donate.jsx [hunk 3]
```diff
@@ -80,9 +80,9 @@ const Donate = () => {
         >
           <h1 className="mb-4">Yes, you help live !!</h1>
           <p>
-            Donations does play an important part as our annual funds, donations
-            from your ends helps thousands of unfortunate people live their
-            lives.
+            Donations do play an important part in our annual funds, donations
+            from your end help thousands of unfortunate people live their
+            lives..
```
Comment (tamalCodes) [inline src/pages/Donate.jsx:85]: Remove this extra dot after `lives.`

File: src/pages/shops/ShopCategory.jsx [hunk 4]
```diff
@@ -25,7 +25,7 @@ const ShopCategory = () => {
 
               <p className="shop_header2">
                 All the products listed here, are made by our members, local
-                charities, NGOs, orphanages. We don't take any profit from the
+                charities, NGOs, and  orphanages. We don't take any profit from the
```
Comment (tamalCodes) [inline src/pages/shops/ShopCategory.jsx:28]: Remove this extra space after `and`

**Review by tamalCodes — CHANGES_REQUESTED** [review]
With the following changes done, we can get this merge. Please always checkout for duplicates, and unwanted spaces.

File: rules/FrontendSetup.md
Comment (HimanshuNarware) [reply rules/FrontendSetup.md:1]: sorry sir it's my mistake I resolve all the errors I have done in the code.

Comment (tamalCodes) [reply rules/FrontendSetup.md:1]: > sorry sir it's my mistake I resolve all the errors I have done in the code.

Hi there, no need to call me sir - just Tamal is fine, I'm still in college. Mistakes are something we all make & learn. 
Don't worry.

**Comment by HimanshuNarware** [comment]
I have made all changes in my code as you said, but what about the code of conduct you said that dont make any changes in this file

**Comment by tamalCodes** [comment]
Hi @HimanshuNarware , yes we don't want any changes now to the Code of Conduct file which you added. It would be better to remove this .

image

File: public/CODE_OF_CONDUCT.md
Comment (tamalCodes) [inline public/CODE_OF_CONDUCT.md:1]: Please do remove this file, as we are working on it - and we won't be having it inside `/public` directory. 
Thanks.

**Review by tamalCodes — CHANGES_REQUESTED** [review]
The rest of the changes looks good to me, with the below mentioned changes done we can finally merge it !

File: public/CODE_OF_CONDUCT.md
Comment (HimanshuNarware) [reply public/CODE_OF_CONDUCT.md:1]: ok

**Comment by HimanshuNarware** [comment]
CODE_OF_CONDUCT REMOVED

**Pull request merged**